### PR TITLE
info added in microservice.yml

### PR DESCRIPTION
**Pull request by rohit121**
Hello,

Micro Service guide schema needs an "info" element that was missing. I have added that so that it validates and runs properly in the OMG UI>

### CODE REVIEW

**Comment by judepereira** [comment]
@rohit121 reduce version to 0.2 please. 1.x would indicate stability, but it isn’t at this time.

File: microservice.yml [hunk 1]
```diff
@@ -1,4 +1,8 @@
 omg: 1
+info:
+  version: 1.0.1
```
Comment (judepereira) [inline microservice.yml]:
```suggestion
  version: 0.2.0
```

Comment (rohit121) [reply microservice.yml]: makes sense

**Comment by rohit121** [comment]
Done @judepereira

**Pull request merged**